oceannavigator/frontend/src/components/CoordInputPanel.jsx

```diff
@@ -62,7 +62,7 @@ export default class CoordInputPanel extends React.Component {
               onChange={e => this.onChange(e.target, DataInput.LATITUDE)}
             />
             <br />
-            <ControlLabel>{_("Longtitude")}: </ControlLabel>
+            <ControlLabel>{_("Longittude")}: </ControlLabel>
```

Comment:
Typo?

Reply:
wow, when you fix a typo with another typo

Reply:
fixed